packages/next/src/build/index.ts

```diff
@@ -1626,6 +1629,7 @@ export default async function build(
             'edge-server',
           ]).then((res) => {
             durationInSeconds += res.duration
+            traceMemoryUsage('Finished edge-server compilation', nextBuildSpan)
```

Comment:
Do we want to grab the memory usage from inside of the worker when using the `webpackBuildWorker` mode?